pom.xml

```
@@ -31,7 +31,7 @@
 
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.414.3</jenkins.version>
+    <jenkins.version>2.462.1</jenkins.version>
```

[Choosing a Jenkins version](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions) recommends 2.440.3 or 2.452.4.  I think that we should prefer 2.440.3 rather than jumping to the most recent LTS.
```suggestion
    <jenkins.version>2.440.3</jenkins.version>
```